whats-new-in-pgstream-0-4-0.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: 'pgstream v0.4.0: Postgres-to-Postgres replication, snapshots & transformations'
```

Comment:
I think the title and the description say the same, thoughts on maybe changing the title to something simpler, like for pgroll updates?
@CezzaineZ @alexfrancoeur 
```suggestion
title: 'pgstream v0.4.0 update'
```

Reply:
As discussed https://xata-hq.slack.com/archives/C05474E6PRQ/p1743072481297669?thread_ts=1742927569.307849&cid=C05474E6PRQ, let's change the description instead.